classes/events/GenerateScooterTrips.py

```diff
@@ -0,0 +1,9 @@
+from classes.events import Event
+
+
+class GenerateScooterTrips(Event):
```

Comment:
What is the point of this class? I don't understand, please help

Reply:
Its an event class that has the purpose to generate new ScooterDeparture and ScooterArrival classes. The purpose of creating all these Event subclasses is to call event.perform() and then the program doesn't care if its a ScooterDeparture,  GenerateScooterTrips, VehicalArrival or ScooterArrival